### PR TITLE
New Improved get all fulfillments from Shopify

### PR DESCRIPTION
**Pull request by Hariador**
New version of the PR since I somehow clobbered my local.

### CODE REVIEW

File: src/Services/Fulfillment.php [hunk 1]
```diff
@@ -20,6 +20,23 @@ public function create(ShopifyFulfillment $fulfillment)
         return $this->unserializeModel($raw['fulfillment'], ShopifyFulfillment::class);
     }
 
+    /**
+     * @param int $OrderId
```
Comment (rickywiens) [inline src/Services/Fulfillment.php]: orderId please.

File: src/Services/Fulfillment.php [hunk 2]
```diff
@@ -20,6 +20,23 @@ public function create(ShopifyFulfillment $fulfillment)
...
+        $raw = $this->client->get("admin/orders/{$orderId}/fulfillments.json", []);
+        $results = collect();
+        foreach ($raw['fulfillments'] as $fulfillment) {
+            $results->push($this->unserializeModel($fulfillment, ShopifyFulfillment::class));
+        }
+
+        return $results;
+    }
+
+
```
Comment (rickywiens) [inline src/Services/Fulfillment.php]: The lint is complaining about this extra line.

Comment (Hariador) [reply src/Services/Fulfillment.php]: Fixed

File: src/Services/Fulfillment.php (hunk 1 above)
Comment (Hariador) [reply src/Services/Fulfillment.php]: Fixed

File: src/Services/Fulfillment.php [hunk 3]
```diff
@@ -20,6 +20,22 @@ public function create(ShopifyFulfillment $fulfillment)
         return $this->unserializeModel($raw['fulfillment'], ShopifyFulfillment::class);
     }
 
+    /**
+     * @param int $orderId
+     *
+     * @return ShopifyFulfillment | Collection
+     */
+    public function get(int $orderId)
```
Comment (rickywiens) [inline src/Services/Fulfillment.php]: Can you remove the typehint here? This breaks compatibility with older php versions

Comment (Hariador) [reply src/Services/Fulfillment.php]: Okay.  There's already type hints in this file though.

**Pull request merged**